src/vehicle.cpp

```diff
@@ -1237,6 +1243,8 @@ void vehicle::use_controls()
         break;
     case control_cancel:
         break;
+    case toggle_scoop:
+        scoop_on=!scoop_on;
```

Comment:
Any prejudice against spaces that it seems like I may have is merely the result of forgetting to add them.

I'll rectify it.